ethpm/package.py

```diff
@@ -115,7 +115,7 @@ def __init__(
         validate_w3_instance(w3)
 
         self.w3 = w3
-        self.w3.eth.defaultContractFactory = cast(Type[Contract], LinkableContract)
+        self.w3.eth._default_contract_factory = cast(Type[Contract], LinkableContract)
```

Comment:
Was this ever meant to be a public property? 🤔

I don't mind putting this back, but this made a bit more sense. We don't have documentation on this afaik.